### PR TITLE
Fix fetch retrieval on browsers

### PR DESCRIPTION
**Pull request by manuquentin**

### CODE REVIEW

**Review by sboily — DISMISSED** [review]

File: src/web-rtc-client.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { C } from 'sip.js/lib/core/messages/methods/constants';
 import { URI } from 'sip.js/lib/grammar/uri';
 import { Parser } from 'sip.js/lib/core/messages/parser';
 import { UserAgent } from 'sip.js/lib/api/user-agent';
-import { stripVideo } from 'sip.js/lib/platform/web/modifiers/modifiers';
+import { stripVideo, holdModifier } from 'sip.js/lib/platform/web/modifiers/modifiers';
```
Comment (sboily) [inline src/web-rtc-client.js:23]: Seems this PR have also fixes for hold. Can you make another PR or change the PR description?

**Review by sboily — CHANGES_REQUESTED** [review]

File: src/web-rtc-client.js (hunk 1 above)
Comment (manuquentin) [reply src/web-rtc-client.js]: Done, thx
